examples/DotNetCore/WebJobs/WebJobHelloWorld/WebJobHelloWorld/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Configuration;
+
+namespace WebJobHelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hostBuilder = new HostBuilder();
+            hostBuilder.ConfigureHostConfiguration((config) =>
+            {
+                LoadConfiguration(config);
+            });
+            hostBuilder.ConfigureWebJobs(b =>
+            {
+                b.AddAzureStorageCoreServices();
+                b.AddAzureStorage();
+            });
+            hostBuilder.ConfigureLogging((context, b) =>
+            {
+                b.AddConsole();
+            });
+
+            var host = hostBuilder.Build();
+            using (host)
+            {
+                host.Run();
+            }
+        }
+
+        private static void LoadConfiguration(IConfigurationBuilder configBuilder)
+        {
+            //
+            // This application attempts to connect to Azure App Configuration to retrieve Azure Blob Storage name, and Queue Name for the Azure Web Job.
+            // It reads the ConnectionString for the App Configuration Service from appsettings.json and from environment variables.
+            configBuilder.AddJsonFile("appsettings.json")
+                .AddEnvironmentVariables();
+
+            var config = configBuilder.Build();
+            if (string.IsNullOrEmpty(config["ConnectionString"]))
+            {
+                throw new ArgumentNullException("Azure App Configuration Connection string not found in appsettings.json or environment variables");
```

Comment:
Can we also include the environment variable name of the connection string we are looking for, similar to what you have in the console app example?